plugins/memcached/CMakeLists.txt

```diff
@@ -25,27 +25,11 @@
 project( memcached )
 
 if (MEMCACHED)
-    ADD_PLUGIN(memcached)
-    if(MAKE_MEMCACHED)
-        # Currently libmemcached versions are not sufficient on ubuntu 12.04 and 14.04 LTS
-        # until then, we build the required libraries from source
-
+    if(NOT DEFINED LIBMEMCACHED_VERSION)
```

Comment:
Even if this is defined at config time it still needs to be checked against min version, right?